src/int.cr

```diff
@@ -254,6 +254,10 @@ struct Int
     end
   end
 
+  def to_hex
```

Comment:
I moved it here to reduce some duplication, but this isn't needed anymore. I'm not sure there is any need for this method in the public API.